### ISSUE TITLE
Schema for webpack config 'externals' field is outdated

### ISSUE THREAD

**Issue by AdirAmsalem** [wix/yoshi#830]
# 🐛 Bug Report

### 😯 Current Behavior

when i use the following `externals` configuration:
```
"externals": {
  ...
  "firebase/app": ["firebase", "app"],
  ...
},
```

i get the following warning from `yoshi`:
```
Warning: Invalid configuration object. Yoshi has been initialised using a configuration object that does not match the API schema.
 - configuration/externals should be a string.
   Field Description: An exact matched dependency becomes external. The same string is used as external dependency.
 - configuration/externals/firebase~1app should be a string.
 - configuration/externals/firebase~1app should be an object.
 - configuration/externals/firebase~1app should be a boolean.
 - configuration/externals/firebase~1app should be one of these:
   string | object | boolean
```

it happens because of the schema defined here: https://github.com/wix/yoshi/blob/master/packages/yoshi-config/schema/yoshi-config-schema.json#L14-L58

### 🤔 Expected Behavior

it should not output any warning for arrays as it's valid and documented here: https://webpack.js.org/configuration/externals/#array
moreover, it’s practically working :)

### 💁 Possible Solution 

probably fixed here: https://github.com/webpack/webpack/pull/8043
so perhaps the schema we use should be updated.

### 👣 Steps to Reproduce 

use the aforementioned externals configuration and it'll show up.

thanks!

**Comment by ronami** [comment]
Thanks for the issue @AdirAmsalem.

I talked with @netanelgilad about it and I think it's best to remove Webpack's schema from our code, and instead `require()` it from `webpack/schemas`. We can construct our own schema dynamically, so we won't have to chase after Webpack with fixes or changes to their schema.

PRs welcome :)

**Comment by ronami** [comment]
Fixed in #941 thanks to @matveyok 🎉
